### PR TITLE
api to expose imported public keys

### LINKED ISSUE #69: [GraphSDK] need new API endpoint to expose imported public key for a user
Since we are avro encoding public keys inside chain for any wallet to import related private keys they need to know what public keys are associated for any user. In that regard we would need a new endpoint that would return imported public keys for a user

### PR DESCRIPTION
**Pull request by aramikm**
# Goal
The goal of this PR is provide an api to expose imported public keys

Closes #69 

# Checklist
- [x] FFI binding added 
- [x] Tests added

### CODE REVIEW

**Review by saraswatpuneet — APPROVED** [review]
Look good! Want to add the same to FFI?

**Comment by aramikm** [comment]
Oops forgot about that. Good point. I'll add

**Review by saraswatpuneet — APPROVED** [review]

File: bridge/ffi/src/c_api.rs [hunk 1]
```diff
@@ -249,6 +249,27 @@ pub unsafe extern "C" fn graph_get_one_sided_private_friendship_connections(
...
+	let result = panic::catch_unwind(|| {
+		if graph_state.is_null() {
+			return DsnpPublicKeys { keys: std::ptr::null_mut(), keys_len: 0 }
+		}
+		let graph_state = &mut *graph_state;
+		let user_id = &*user_id;
+		let keys = graph_state.get_public_keys(user_id).unwrap_or_default();
+		let ffi_keys = dsnp_public_keys_to_ffi(keys);
+		let keys_len = ffi_keys.len();
+		let keys_ptr = ManuallyDrop::new(ffi_keys).as_mut_ptr();
```
Comment (JoeCap08055) [inline bridge/ffi/src/c_api.rs:267]: Don't we also need a method to manually release this memory at some point?

Comment (aramikm) [reply bridge/ffi/src/c_api.rs:267]: We should, should we do that in this layer or that needs to be done from the layer that consumes FFI?

Comment (JoeCap08055) [reply bridge/ffi/src/c_api.rs:267]: The layer that consumes the FFI will need to invoke it, but we still need to provide a native Rust method to release the memory. For instance, C or Java code can't natively de-allocate memory that was allocated in Rust. So I think we will need a `free_public_keys()` function.

Comment (aramikm) [reply bridge/ffi/src/c_api.rs:267]: I'll take care of it in a future PR in that case.

**Pull request merged**